components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/OAuth2ScopeService.java

```diff
@@ -823,4 +829,34 @@ private UserApplicationScopeConsentDO getConsentsToBeAdded(UserApplicationScopeC
         consentToBeAdded.setDeniedScopes(disapprovedScopes);
         return consentToBeAdded;
     }
+
+    @Override
+    public String getName() {
+
+        return "OAuth2ScopeService";
```

Comment:
What's the usage of this name?

Reply:
Used in debug logs to log the Scope service name

Reply:
Can't we use the class name for that?